\section{Methods}

We sequenced one wild individual each of Orthochromis sp. `red cheek,' {\em Serranochromis} sp. `checkerboard,' {\em Paralabidochromis paucidens}, {\em Harpagochromis vittatus}, {\em Paralabidochromis flavus} and {\em Harpagochromis} sp. `checkmate.' We used PCR-free methods \cite{kozarewa2009amplification} to generate paired-end reads using Illumina HiSeq. We generated 50-65 million reads per fish -- approximately 10-fold coverage -- with the exception of {\em Harpagochromis} sp. `checkmate,' which was sequenced at higher depth for a different project (Neches {\em et al.} in preperation). We randomly reduced the number of raw {\em Harpagochromis} sp. `checkmate' reads to the level of the other five species. We then aligned all six to the {\em Oreochromis niloticus} ({\tt oreNil2}) genome \cite{brawand2014genomic} using {\tt bowtie2} \cite{langmead2012fast} and {\tt samtools} \cite{li2009sequence}. We then called all confident sites with {\tt GATK unifiedgenotyper} \cite{mckenna2010genome}.

\subsection{Assembling the mitogenome and building a mitogenome tree}

In order to establish phylogenetic relationships among the six taxa and to provide an estimate of genetic divergence that is independent of our nuclear genomic data, we extracted and assembled mitogenome data from the reads of each species utilizing the {\tt mitobim} \cite{hahn2013reconstructing} pipeline and the {\em Oreochromis niloticus} mitogenome. We then aligned reads with {\tt muscle} \cite{edgar2004muscle}, and manually trimmed each mitogenome to a size of 16,669bp so that all species had sequence of the same base pair length. We compared several substitution models using {\tt jmodeltest} 2.1.7 \cite{darriba2012jmodeltest}, finding that a {\tt GTR+G} model was the best fit to the data. We then used {\tt beast} 2.2.1 \cite{bouckaert2014beast} with default parameters to infer a phylogenetic tree for our six mitochondrial genomes, using {\tt tracer} v1.8 \cite{rambaut2003tracer} to assess performance and {\tt treeannotator} with 25\% burn-in to produce a consensus mitogenome tree (Figure \ref{UL_fig1}). Because of the controversy associated with appropriate calibrations of the cichlid molecular clock \cite{genner2007age, azuma2008mitogenomic, friedman2013molecular, schwarzer2015phylogeny}, we compare only relative ages here.

\subsection{Genomewide levels of divergence}

We then calculated patterns of coverage, called SNPs, $d_f$ (fixed divergence) and $D_{XY}$ within 10-kb blocks across all the 22 linkage groups of our reference for each lake pair. We used custom {\tt R} and {\tt Python} scripts, all of which are compiled into a Jupyter notebook \cite{perez2007ipython}. 10-kb blocks where only <80\% of all bases were sequenced for at any one of the fish were excluded from further analyses, leaving 39 207 10-kb blocks, over 66\% of the sequence of the entire genome.

$d_f$ and $D_{XY}$ values were calculated between the ecologically divergent species of each lake, and compared in several ways. A given site was assessed as fixed divergent when both individuals of a given lake are fixed for alternative alleles, but because we only use one individual per species, some sites recorded as divergent would be classified as variable if several individuals within a species were used. Therefore, our estimates of fixed divergence are merely an upper bound on the true number of strongly divergent sites.

We first compared overall patterns of fixed divergence between Victoria, Kivu and Mweru with a Kruskal-Wallis test (Figure \ref{UL_fig1}, Table S1, Supporting information). We then compared whether genomewide fixed divergence in a block ever exceeds the mean plus two times the standard deviation for each lake pair (Table S1, Supporting information). We also compared fixed divergence in outlier regions from the two younger ecomorph pairs to genomewide divergence in the older ecomorph pair using a Kruskal-Wallis test. We compare the top 1\%, 0.1\% and 0.01\% outliers (Table S1, Supporting information).

Establishing whether outlier regions for relative measures of differentiation, like $d_f$, also exhibit overall higher absolute sequence divergence is critically important in establishing whether divergent selection in the face of gene flow was likely to have played a role in speciation and species divergence \cite{charlesworth1998measures, cruickshank2014reanalysis}. Therefore, we compared absolute divergence $D_{XY}$ between regions with elevated $d_f$ vs the rest of the genome for multiple outlier levels (Figure \ref{UL_fig3}, Table S2, Supporting information). In order to test whether parallel phenotypic divergent evolution is associated with a signature of parallel genomic divergent evolution between the three species pairs from different lakes, we also compared whether regions exhibiting high $d_f$ levels in a given lake pair also exhibited high levels of $D_{XY}$ in the other pairs (Figure \ref{UL_fig4}, Table S2, Supporting information).