\section{Discussion}

Some form of co-diversification is probably inevitable in situations where multiple species remain in ecological contact over evolutionary time. Most of the literature on the topic addresses the co-diversification in one of two ways, depending on the initial assumptions :

\begin{enumerate}
	\item {\em Given a system in which co-diversification may have occurred, what is the likelihood that it did?}

	\item {\em Given that co-diversification is known to have occurred, what is the most likely history?}
\end{enumerate}

We are interested in learning about the natural history of the system, and so we would like to ask a somewhat enlarged question. {\em Given a system in which there are many opportunities where co-diversification may have occurred, which cases are the most likely?} 

\subsection{Phylogenetic signal and trait-based approaches}

For each organism that appears among the microbioms of a group of hosts, there exists a pattern of observed associations between the ``guest'' organism and the host organisms. These associations can be treated as traits of the host, and the history of each association can be examined like any phenotypic trait. Alternatively, the tendency to associate (or not) with a host could be treated as a trait distributed over a group of microbial organisms, although it is difficult to imagine how one might demarcate the boundaries between groups. Association with a particular host is more likely to correspond to different traits for more distantly related bacteria, and so it is probably not meaningful to naively apply stochastic trait mapping across all bacterial diversity.

In {\em Analysis of phylogenetics and evolution with {\tt R}} (p. 236), Emmanuael Paradis describes phylogenetic signal as follows :

\begin{quote}
The concept of phylogenetic signal is at the heart of most phylogenetic methods. From a statistical point of view, a phylogenetic signal is defined by the non-null covariances (i.e., non-independence) among species. From a biological point of view, phylogenetic signal is a direct consequence of the evolution of traits and its form will depend on the evolutionary mechanisms in action.
\end{quote}

The advantage of this approach is that it decomposes the problem into a large number of smaller problems. Every OTU is treated as an independent trait, and observed associations are simply plugged into one of the four established models. All of them have been implemented in {\tt R}. Unfortunately, this is also a key disadvantage. Most trait-based models do not account for interactions {\em among} traits, or a way to account for traits that have their own evolutionary model, independent of the organisms that exhibit them. 

A second disadvantage of trait-based approaches when applied to co-diversifying systems arises from the non-uniform distribution of microbial diversity with respect to 16S diversity. Individual OTUs may represent several organisms, each with their own pattern of association (Figure \ref{fig:FP_fig4}). Comparing phylogenetic signal among different OTUs thus depends on the diversity underlying each of the OTUs. Correcting for this would require a relatively complete and unbiased survey of microbial diversity.

\subfile{FishPoo/figure4}

\subsection{Tests for cospeciation}

In what could now be considered a classic paper, Hafner {\em et al.} \cite{hafner1994disparate} examine the relationship between pocket gophers and their chewing louse parasites. This paper addresses there first question using a tree reconciliation test implemented in {\em COMPONENT} by Roderic D. M. Page \cite{page1993genes}. It is, in essence, a parsimony approach; the gene duplication and loss events necessary to achieve topological congruence between two trees are minimized. Hafner {\em et al.} first reject the null hypothesis (independent evolution) by applying the tree reconciliation test among gopher, louse and randomly drawn trees, and then examine rates of nucleotide divergence between the gopher and louse. 

This dataset has since been reanalyzed in many other publications, has become a sort of base-case for co-diversification methods. Notably, Huelsenbeck {\em et al.} introduced the use of Bayesian inference to cospeciation \cite{huelsenbeck2000bayesian}.

Hommola {\em et al.} \cite{hommola2009permutation} describe a method extends the Mantel test. \cite{mantel1967detection} The Mantel test is a statistic on the correlation of two matrixes, and requires that the matrixes be of equal rank. When applied to distance matrixes for trees containing differing numbers of taxa, one must delete or duplicate taxa until the matrix ranks are equal. This results in anomalous results which Hommola {\em et al.} explore in detail. To address this issue, they take every pair of linked tips (e.g., a parasite species that is observed to associate with a host species), and examine the correlation of distances through the two trees. This accommodates trees of arbitrary size and arbitrary patterns of association among their tips, and can be coupled with a straightforward permutation test to estimate the significance of the correlations. One can walk through every clade in the phylogeny of host-associated OTUs and, every clade, compute the Hommola correlation with the host phylogeny (Figure \ref{FP_hommola_corr}). 

\subfile{FishPoo/figure5}

However, there are some important difficulties when it comes to applying the Hommola cospeciation test to a large number of clades belonging to the same tree. Fist of all, this approach necessarily entails multiple comparison, and so a correction to the significance values is required. However, a Bonferroni correction is not appropriate because the structure of each clade is not independent of the others. Rather, they are hierarchically nested within a tree, which is itself inferred from an probabilistic model (an approximate maximum likelihood model, in this case), which is based on nucelotide transitions inferred from from an alignment. The autocorrelation within the tree would need to be accounted for, but calculating it would not be straightforward. At a minimum, one would need to take into account the fact that the multiple comparisons have hierarchical relationships.

The second difficulty arises from the need to perform unsupervised correlation tests. Hommola {\em et al.} use the Pearson product-moment correlation, which assumes that the processes are independent, identically distributed, and follow a bivariate normal distribution. Unfortunately, there is reason to suppose that the distances among tips of a phylogenetic tree would be normally distributed. Of course, other correlation statistics could be substituted at the expense of computational complexity (Spearman's rank correlation coefficient, for example, obeys quadratic scaling), but the number OTUs present in a typical microbiome makes this impractical.

At root, though, there are many subtle ways that summary statistics can be misleading. As a discipline, statistics warns people against drawing conclusions from them without examining the structure of the data. \cite{anscombe1973graphs} For example, compare the distribution of pairwise distances for the Gopher/Louse dataset \cite{hafner1994disparate} to Clade 72223, which appears just below it in Figure \ref{FP_highcorr}. Both have about the same correlation coefficient ($r=0.490$ verses $r=0.488$), very high significance ($1.38\times 10^{-9}$ verses $p=3.21\times 10^{-223}$ and are within an order of magnitude in size ($15 \times 17$ vs $14 \times 68$). Nevertheless, the two distributions are obviously different, and the vertical banding pattern in Clade 72223 is strongly reminiscent of Case 4 in Anscombe's quartet. 

\subfile{FishPoo/figure6}

% At root, however, there is a trade-off between statistical robustness and interpretability at work