\section{Introduction}

Ecology is the study of the relationships among organisms and between organisms and their environment. When making observations in a macroscopic system, a vast amount of information is available in plain sight to a trained observer. Suppose, a field bulgiest is taking observations of a specific feature of an interaction between an mammal and a parasitic louse in the field. Even if the scope of the observational protocol is defined narrowly, other potentially important information is nevertheless available. Behavior of the host and parasite will be displayed. Aspects of life cycles may appear. Abiotic properties of the habitat, such as elevation and physical structure, will be known. Ecological connections with other organisms outside the scope of the host-parasite interaction in question may present themselves. These pieces of data provide the observer with a great deal of context about the system they are examining, and may help the identify weaknesses in observational protocol, in the overall study design or in the hypothesis itself. Often, such contextual information only finds its way into rigorous experimental design by helping to frame and focus the hypothesis. 

% I hope this illustrates the idea more clearly now. I'm interested the kind of information that is often NOT explicitly mentioned anywhere in the results that people publish. For example, suppose you have a hypothesis like, "The role of tails in sexual displays of birds and mammals is an example of convergent evolution." To frame a hypothesis like that, at some point you probably had to have a thought like, "Huh. I just noticed that birds and mammals both have tails."

% If you're missing this sort of information, it's easy to come up with totally uninformative frames that look perfectly reasonable.

% I'm trying to argue that in microbiology, this information doesn't fall into your lap in the way it often does in other areas of biology. Visual observations are interesting and important, but because you can't connect visual observations to taxonomy very well, they don't play the same role in microbiology that they do in other areas. Instead of being able to say ``birds and mammals both have tails,'' you would have to say, ``A lot things have tails.'' You can't get to the idea about convergent evolution from there.

% I know this point seems pretty abstract, but I want to argue that the tool I've built helps bring some of this ``accidental'' knowledge to microbial ecology. 

While is well established that microorganisms exist in complex relationships with their environment and with one another, for the overwhelming majority of microorganisms known to science, our only knowledge is the simple fact of their existence. As with macroscopic communities, one can also make non-targeted observations of microbial communities using microscopy and environmental sequencing. However, one often cannot cross-reference or link these observations in the intuitive way that is usually possible in macroscopic communities. In a macroscopic community, the link between molecular data and observations of morphology, behavior and life history is an inevitable consequence of the need to physically collect the sample from individual specimens. In microbial communities, establishing such links is painstaking, deliberate work, and is often impractical. 

Single-cell sequencing may help to address this difficulty, but it remains to be seen how generally it can be applied as a observational tool (to say nothing of the expense). Even so, there are other complications that tend to be much more pronounced in microscopic communities. Metabolically intimate relationships are often separated over vast distances and long intervals, and mediated by substances that cannot be observed optically and require sensitive, targeted chemical analysis to measure. Convergent evolution often drives distantly related organisms to present identical morphology, and closely related organism frequently present highly divergent morphology. The great diversity of microorganisms routinely outstrips our ability to collect, organize and interpret meaningfully specific metadata.

The result is that knowledge of microbial communities tends to be much more fragmented than macroscopic communities, even when the total amount known is roughly the same. Key insights into microbial ecology are often obscured by methodological ambiguities that confound cross-referencing different categories of observational data. 

However, different types of ecological interactions impose distinct patterns on the evolution of the organisms they involve. The evolution of organisms in predator-prey, host-parasite and other antagonistic interactions can be modeled with Red Queen dynamics \cite{van1973new}, where reciprocal adaptive responses drive cospeciation. Mutualistic interactions tend to obey different dynamics. The Red King model, \cite{bergstrom2003red, gokhale2012mutualism} for example, predicts that in mutualistic interactions, advantages tend to accrue to slower-evolving organisms. This has the effect of reinforcing the development of complex, highly nested interactions. \cite{bastolla2009architecture, rezende2007non } This is by no means an exhaustive list of types of interactions or the evolutionary dynamics they drive. If ecology can drive evolutionary processes, then it ought to be possible to infer the nature of an ecological interaction from the signature it imprints on the evolution of the organisms involved. Such inferences are potentially of great value for understanding microbial communities. Usually, the phylogeny and the origin are the only pieces of data available for the bulk of observed biodiversity.

In this study, we apply techniques from graph theory to construct a high dimensional space representing structural features of networks of interacting host and microbe phylogenies. Using a database of interactions with know ecologies gathered from the literature, we use standard machine learning techniques to make predictions about the ecological roles of microbial clades in our system. This approach will work for any system where the microbial community ramifies among several host species, but it may be easily generalized to any system where rate of dispersal through the system is similar to the rate of microbial phylogenetic divergence that can be detected. 

The habitat we have chosen to illustrate this method is a group of 14 species of Tanganyikan cichlid fish. With the exception of {\em Tylochromis polylepis} and {\em Oreochromis tanganicae}, the Tanganyikan cichlids are a species flock of more than two hundred described species sharing a common ancestor dating to the formation of the lake between ten and twenty million years ago. While some members of this species flock are older than hominids, other lineages have undergone much more recent speciation, including some in which the process of adaptive radiation appears to be ongoing. Within this species flock, a large variety of reproductive, defense and trophic strategies have evolved. Other than the anoxic abyssal region, cichlids have colonized every habitat in the lake and occupy every trophic level above primary producers. Taken together, this species flock represents roughly 0.5\% of all described fish species and 0.25\% of all vertebrate species.

