\section{Introduction}

The dynamic modulation of gene expression is an important mechanism that allows organisms to sense and respond to changes in their environment. These changes in expression profiles are mediated by dynamic associations of transcription factors and their cognate regulatory regions, collectively known as gene-regulatory networks (GRNs) \cite{davidson2001}. Regulatory networks integrate complex cellular and environmental cues, orchestrating intricate phenotypes essential for physiology and development. The evolutionary rewiring of these regulatory circuits is thought to be an important driver of speciation \cite{shou2011measuring}. Elucidating the structure and function of GRNs is therefore a major research initiative in functional genomics and systems biology \cite{harbison_transcriptional_2004, peter_modularity_2009, kaleta_integrative_2010, palaniswamy_agris_2006, fadda_inferring_2009, bonneau_predictive_2007}.

The characterization of GRN architecture has been driven by advances in experimental and computational methods for identifying genome-wide proteinâ€“DNA interactions \cite{hesselberth_global_2009, park_chip-seq:_2009, wilbanks2010evaluation, bonneau_inferelator:_2006, de_jong_modeling_2002}. One such approach is chromatin immunoprecipitation (IP) coupled with high-throughput sequencing (ChIP-seq), a method that provides quantitative genome-wide mapping of target protein-binding events. ChIP-seq identifies protein-binding sites with improved spatial resolution and decreased cost relative to previous microarray-based ChIP-chip technologies \cite{park_chip-seq:_2009}. While ChIP-seq has become a widely used tool in eukaryotic systems, this method has been applied only once in a bacterial system \cite{lun_blind_2009} and there exist no instances of such work in archaea. The small size of bacterial and archaeal genomes makes this high-throughput sequence technology particularly attractive, as sample multiplexing can be used to dramatically reduce costs relative to microarray-based platforms.

Developing a ChIP-seq protocol for archaea would stimulate high-throughput characterization of GRNs, which are a nascent area of study relative to work in the other two domains of life. Archaea are essential drivers of global biogeochemical cycling, integral players in industrial applications and biomedically important organisms. Furthermore, the transcriptional apparatus of archaea exhibits properties of both eukaryotic and bacterial systems, making it an intriguing target for investigating basic principles of regulatory mechanisms across the tree of life \cite{bell_archaeal_2005}. Improved understanding of archaeal information processing and transcriptional regulation has widespread applicability.

We present a novel ChIP-seq workflow for the archaea using the model organism {\em Halobacterium salinarum} sp. NRC-1 ( {\em Hb.} NRC-1 ) and demonstrate its application for mapping the genome-wide binding sites of natively expressed transcription factors. Previous bacterial and archaeal ChIP methods have taken different approaches involving either costly protein-specific antibodies against native proteins \cite{lun_blind_2009} or a standard antibody against epitope-tagged target proteins that are constitutively overexpressed from a heterologous plasmid \cite{facciotti_general_2007, kaur_coordination_2010}. This protocol combines these methods by employing a single, commercially available antihemagglutinin (HA) antibody against natively expressed recombinant target proteins. This ChIP-seq method maintains sensitivity and specificity with as little as $\approx 1$ ml of the typical bacterial or archaeal culture, making it suitable for high-throughput analyses. Multiplexing of samples during sequencing significantly decreases experimental costs relative to previous ChIP-chip methods, without diminishing sensitivity and specificity.

A complimentary bioinformatics method is presented for user-friendly identification of binding events using the Pique python package. Integration with the Gaggle toolkit streamlines the exploration and analysis of putative protein-binding sites \cite{bare_integration_2010, shannon2006gaggle}. This end-to-end workflow for ChIP-seq of natively expressed proteins provides a suitable platform for large-scale studies of the structure and dynamic remodeling of GRNs. The first protocol of its kind for archaea, this method can be adapted for work in all bacteria and archaea with suitable genetic tools.
