\section{Materials \& Methods}

\subsection{Sample collection}

Samples were collected from built environment surfaces throughout the United States on cotton swabs (Puritan 25-806 2PC) and mailed (usually overnight) to the University of California, Davis where they were transferred to lysogeny broth (LB) plates. Colonies were chosen for further examination based on maximizing morphological variation. Each chosen colony was double-dilution streaked (two rounds of streak plates) and then the identity determined by direct PCR and Sanger sequencing using the {\tt 27F} ({\tt 5'-AGAGTTTGATCMTGGCTCAG-3'}) and {\tt 1391R} ({\tt 5'-GACGGGCGGTGTGTRCA-3'}) primers (see Dunitz et al., 2015 for details). Sanger sequences were trimmed and aligned using {\tt Geneious}. \cite{kearse_geneious_2012} The resulting consensus sequence was identified through a combination of {\tt BLAST} \cite{altschul_basic_1990} and building phylogenetic trees using the Ribosomal Database Project (RDP) website. \cite{cole_ribosomal_2014} The 48 candidates for spaceflight were chosen on the basis of biosafety level (BSL-1 only), taxonomic variety, and human interest. In the absence of international standards, the biosafety level of each organism was determined by searching the American Biological Safety Association (ABSA) risk group database, the American Tissue Culture Collection (ATCC), the {\em Deutsche Sammlung von Mikroorganismen und Zellkulturen} (DSMZ), and other public databases. An organism was removed from consideration if it was listed as BSL-2 or higher in any country or collection in the world. Human interest was an arbitrary set of criteria such as unusual physiology, catchy name, or a memorable original isolation source.

\subsection{Growth experiment}

A set of bacterial plates were created for each aspect of the study: growth in microgravity on the ISS (space plates), or growth on Earth (ground plates). The plates were created using clear agar to facilitate optical density (OD) measurements. 1.5 grams of Gelzan CM agar (Sigma-Aldrich, St. Louis, MO, USA) was added to 1 liter of lysogeny broth (LB). Each well of a flat-bottomed 96-well plate (Costar, Corning, NY, USA) was plated with 200 \si{\micro\liter} of agar. The plates were flamed to remove bubbles and incubated for 48 to 72 hours at room temperature ($\approx 20$\degree C) to ensure sterility before adding bacteria. Fresh overnights of each bacterial isolate were diluted to 0.01 OD600 and made into 8\% glycerol stocks. For plating, 10 \si{\micro\liter} of each thawed stock dilution was added to two wells per 96 well plate. Six replicate plates were made. The bacteria were placed into different locations on each plate in order to account for drying at the edges or any other positional effects on the plates. The plates were then sealed with adhesive polypropylene film (VWR \#60941-072), into which a grid of micron-diameter holes were cut with a laser to allow for airflow. The ground plates were stored at -80\degree C at UC Davis, and the space plates were mailed on dry ice to the National Aeronautics and Space Administration (NASA) Johnson Space Center in Houston, TX before transfer (at -80\degree C) to Cape Canaveral, FL for launch.

This payload was flown on the CRS-3 launch of the Space Exploration Technologies (SpaceX) Dragon spacecraft, on a Falcon 9 v1.1 rocket which successfully launched April 18, 2014. After six days, the space plates were removed from the MELFI (Minus Eighty Lab Freezer For ISS) and partially thawed. However, technical problems arose and the space plates were placed back into the MELFI until December 8, 2014. At that time, all three plates were thawed and the OD600 of each well ($3\times 3$ grid) was measured at time 0 (60 minutes after removal from the freezer) and then every 24 hours for four days. Measurements were performed in a Molecular Devices SpectraMax M5e plate reader which was modified for integration onto the ISS. On these same days, equivalent measurements of the ground plates were taken in a Molecular Devices SpectraMax M5e plate reader at UC Davis. The exception to this was the initial partial thawing, which was not replicated with the ground plates since the amount of thaw was not reported by the astronauts. After the experiment, the ground plates were placed back at -80\degree C and the space plates were placed back into the MELFI. In February 2015, the space plates were transferred to a -95\degree C freezer on board a Dragon spacecraft. The vehicle splashed down in the Pacific Ocean on Feb 10, 2015. The space plates were then mailed to UC Davis on dry ice and were transferred to -80\degree C when received.

Once the plates arrived, we thawed all six plates and performed a high-density measurement in a Tecan M200 plate reader. OD600 readings were taken in a $5\times 5$ grid covering the entire well, these 25 measurements were then averaged within each well.

\subsection{Analysis}

For each sample, the averages of the six space replicates and six ground replicates were compared using a Studentâ€™s t-test. To correct for multiple hypothesis testing, the p-values were adjusted using the False Discovery Rate (FDR) method. \cite{benjamini_controlling_1995} All raw data, analyses and scripts can be found at \url{https://zenodo.org/record/44661}.

\subsection{Confirmation}

In order to confirm that the observed results were not due to contamination of the wells, each of the 12 replicates (six space, six ground) for the three bacteria showing statistically different growth between the ISS and Earth were cultured after the experiment. Bacteria were struck from the wells onto LB-agar plates, then single colonies were grown into overnight cultures. DNA was extracted using a Wizard Genomic DNA Purification kit (Promega, Madison, WI, USA) from each of the 36 cultures (3 bacteria $\times$ 12 replicates) and the identity was confirmed by BLAST of the Sanger sequenced PCR product using the {\tt 27F} and {\tt 1391R} primers as described above.

\subsection{Comparison to ISS swab data}

The bacterial community on the ISS was recently surveyed by PCR amplification and sequencing of 16S rRNA genes from swabs (Lang {\em et al.}, unpublished data). We compared the 16S sequence of each of our bacterial isolates to the ``representative sequence'' from each operational taxonomic unit (OTU) generated from the survey data. A {\tt BLASTN} search was performed locally and a match was considered to be present in the data when there was 97\% identity over at least 250 bp of the rRNA sequence (the amplified fragment is 253 bp).
