%\noindent\textbf{Part 1: Fish.} 
%This section is made up of three chapters exploring the evolution and ecology of cichlid fish and the relationship between cichlids and the organisms that make up their associated microbiomes. 

%\begin{quote}
\noindent\textbf{Chapter 1.} We propose a method for predicting the ecological function of host-associated
microbes using neural networks trained on a feature space of labeled
ecological interactions from the literature. The feature space is constructed
over the Laplacian spectral density distributions of the networks formed by
linking the phylogenies of the host and microbial clades through their
ecological associations. A classifier trained on 51 interactions with known
ecology and 100 simulated controls was used predict the ecological function in
the microbimes of 14 species of wild-caught cichlid fish from the Lake
Tanganyika adaptive radiation observed using 16S rRNA sequencing.

%\end{quote}

%\begin{quote}
\noindent\textbf{Chapter 2.} Genomic patterns of divergence are examined using whole-genome resequencing of
three sympatric cichlid species pairs with similar functional and ecological
differentiation but different ages, revealing a signature of genomic
divergence. Regions of
elevated relative differentiation exhibit increased absolute
differentiation. We detect a signature of convergent evolution across all
three species pairs. Our results suggest that functional phenotypic
differentiation is associated with a signature of genomic
divergence.

%\end{quote}

%\begin{quote}
\noindent\textbf{Chapter 3.} We show that evolutionary innovations can result in competitive inferiority
and extinction. The modified pharyngeal jaws of cichlid fishes and several
marine fish, a classic example of evolutionary innovation, are not
universally beneficial. Analysis of dietary evolution across marine fish
lineages reveals that the innovation compromises access to
predator niches. This competitive inferiority shaped the adaptive
radiation of cichlids in Lake Tanganyika and played a pivotal, previously
unrecognized role in the mass extinction of cichlid fishes in Lake Victoria
after Nile perch invasion.

%\end{quote}

%\noindent\textbf{Part 2: Oceans.}
%This section is made up of three chapters exploring the biogeography of the planktonic microbial ecology of the open ocean. %Chapter four develops Non Negative Matrix Factorization, a statistical approach for predicting ecological functions of protein-coding genes observed from metagenomic data and applies this approach to the 2004--2007 Global Ocean Survey. \cite{jiang2012functional} Chapters five and six present results from a microbial survey of the Indian Ocean, and develop a citizen science approach for performing oceanographic research using small sailboats. \cite{jeffries2015spatially, lauro2014common}

%\begin{quote}
\noindent\textbf{Chapter 4.} We map protein sequences from the Global Ocean Survey to protein families and
use non-negative matrix factorization (NMF) to approximate linear combinations
of ecological components with characteristic functional and site profiles. We
identify functional signatures, estimate functional distance between sites,
and find that an NMF-filtered measure is more strongly correlated with
environmental distance than a comparable PCA-filtered measure. We find that
functional distance is more strongly correlated with environmental distance
than geographic distance in agreement with prior studies.

%\end{quote}

%\begin{quote}
\noindent\textbf{Chapter 5.} We describe the unique technical, logistical, organizational, and ethical
issues from the 2013 Indigo V Indian Ocean Expedition research cruise from
Cape Town, South Africa, to Phuket, Thailand aboard the S/Y {\em Indigo V}. An
inventory the surface water population of bacterioplankton was collected and
basic measurements of ocean physics and chemistry were tabulated.

%\end{quote}

%\begin{quote}
\noindent\textbf{Chapter 6.} We report on the microbial diversity across the Indian Ocean and a lagoon in
the Chagos Archipelago. The community within the lagoon differed from adjacent
community despite constant water exchange, driven by photosynthetic
cyanobacterium {\em Synechococcus}. Enrichment of photosynthesis-related
transcripts and nutrient cycling indicate influence of primary production on
community structure. A five-fold diurnal increase in viral transcripts within
the lagoon suggests concomitant bacteriophage influence.

%\end{quote}

%\noindent\textbf{Part 3: ChIP-seq.}
%This section is made up of two chapters that develop tools for exploring the gene regulatory networks of halophilic archaea. %Chapter seven lays out a workflow for identifying protein-DNA interactions of natively expressed transcription factors in the model organism {\em Halobacterium salinarum} sp. NRC-1 using ChIP-seq, and has been previously published. \cite{wilbanks2012workflow} Chapter nine presents the software developed for this workflow, and has been published as preprint. \cite{neches2014fit}

%\begin{quote}
\noindent\textbf{Chapter 7.} We present a novel ChIP-seq workflow for archaea using {\em Halobacterium
salinarum} sp. NRC-1 and map binding sites of natively expressed transcription
factors. Relative to ChIP-Chip and qPCR, it improves spatial resolution and
reduces cost. %\end{quote}

%\begin{quote}
\noindent\textbf{Chapter 8.} Pique is a user-friendly, freely licensed ChIP-Seq peak finding application
for bacterial and archaeal ChIP-Seq experiments. Output is provided in
standardized file formats for manual curation and data exploration. %\end{quote}

%\noindent\textbf{Part 4: Tools.}
%This section is made up of two chapters that develop applications for 3D printing in microbiology and molecular biology. %Chapter nine establishes that fused deposition modeling 3D printing is alone sufficient to produce sterile components, and has been previously published. \cite{neches2016intrinsic} Chapter ten presents single-use 3D printed parts, and the software to create them, that can perform the role of a liquid handling robot.

%\begin{quote}
\noindent\textbf{Chapter 9.} With appropriate handling, 3D printers produce sterile
components from non-sterile thermoplastic feedstock without post-fabrication
treatment.
%\end{quote}

%\begin{quote}
\noindent\textbf{Chapter 10.} We present a method for fabricating single-use microtiter plates with volumes
calibrated for each sample, allowing the use of multichannel pipettes for
general liquid handling operations. Many custom plates can be 3D printed
simultaneously, resulting in substantial savings in cost and time.
%\end{quote}

%\noindent\textbf{Part 5: Space.}
%This section is made up of a single chapter that relates a basic microbial growth kinetics experiment aboard the International Space Station. 

%\begin{quote}
\noindent\textbf{Chapter 11.} The growth kinetics of 48 strains of building-associated
bacteria were measured aboard the International Space Station. One strain, {\em Bacillus safensis}
JPL-MERTA-8-2, grew 60\% better in microgravity.
%\end{quote}
