\section*{Introduction}

Twenty years ago, it was hoped that culture-independent sequencing would be the key to understanding how organisms function in microbial ecosystems, and indeed it has revolutionized our understanding of microbial diversity and its dynamics and distribution. However, ecosystem function has remained a difficult problem for two reasons. First, there has been relatively little improvement in our ability to predict a gene's function from its sequence. Second, it remains extremely difficult to measure a gene's activity in a culture-independent manner. Metagenomes are surveys of potential, mostly unknown function.

In retrospect, this is not surprising; for macroscopic systems, it is usually impossible to learn anything about ecosystem function from any composite of extremely small details. An inventory of myoglobin sequences won't reveal whether bears typically eat salmon or salmon typically eat bears. However, knowledge of how the physiology of bears and salmon drives their ecosystem roles {\em does} reveal interesting insights on myoglobin. Sound reasoning must draw inferences {\em from} areas of understanding {\em to} areas of ignorance. We know very little about very small number of molecular mechanisms, and so most sequence data makes a poor anchor from which to begin building chains of inference. Rather, its value is often as a target one may reason {\em towards}. The dream of molecular biology and molecular ecology is a sort of epistemological hysteresis; macroscopic observations form the foundation for inferences on molecular features, which then form the foundation for inferences on macroscopic features, and back again, weaving a balustrade across the gap of scale and scope. As yet, there are but few such bridges, and so the sequencing revolution has drowned us in a vast effusion of answers to questions we do not know how to ask.